pkgs/servers/http/nginx/modules.nix

```diff
@@ -66,6 +70,61 @@ in
 
 let
   self = {
+    acme = rec {
+      name = "acme";
+      src =
+        let
+          src = fetchFromGitHub {
+            name = "acme";
+            owner = "nginx";
+            repo = "nginx-acme";
+            rev = "v0.2.0";
+            hash = "sha256-as+5cdwzpM4tvCxcgCIr877VJf4DduNTGRg29vChDnM=";
+          };
+          combined =
+            runCommand "vendored-repo"
+              {
+                nativeBuildInputs = [
+                  rustPlatform.cargoSetupHook
+                ];
+                cargoDeps = rustPlatform.importCargoLock {
+                  lockFile = "${src}/Cargo.lock";
+                };
+              }
+              ''
+                mkdir -p $out
+                cp -r ${src}/* $out/
+
+                runHook postUnpack
+                cp -r cargo-vendor-dir $out/
+                cp -r .cargo $out/
+              '';
+        in
+        combined;
+
+      preConfigure = ''
+        export NGX_RUSTC_OPT="--config ${src}/.cargo/config.toml"
+        export OPENSSL_NO_VENDOR=1
+      '';
+
+      inputs = [
+        openssl
+        cargo
+        rustPlatform.bindgenHook
+        #rustPlatform.cargoSetupHook
+        #rustPlatform.maturinBuildHook
+        rustc
+        pkg-config
+      ];
+
+      meta = with lib; {
+        description = "An NGINX module with the implementation of the automatic certificate management (ACMEv2) protocol";
```

Comment:
I'd propose to shorten this to "Implementation of the automatic ...", to be consistent with the other Nginx modules (which also don't mention the fact that they are a Nginx module in their description)